app/lib/navigation/state_file_nj_question_navigation.rb

```diff
@@ -30,34 +30,35 @@ class StateFileNjQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantRentPaidController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantPropertyTaxWorksheetController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjIneligiblePropertyTaxController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjEstimatedTaxPaymentsController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjEitcQualifyingChildController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjSalesUseTaxController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjGubernatorialElectionsController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::PrimaryStateIdController),
-                                          Navigation::NavigationStep.new(StateFile::Questions::SpouseStateIdController),
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjYearOfDeathController), # Line 5
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjCountyController), # header
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjMunicipalityController), # header
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjDisabledExemptionController), # Line 8
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjVeteransExemptionController), # Line 9
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjCollegeDependentsExemptionController), # Line 13
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjDependentsHealthInsuranceController), # Line 14
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjMedicalExpensesController), # Line 31
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjEitcQualifyingChildController), # Line 58, intentionally moved up to be in the context of other credits and deductions, and to ensure there is a consistent page after the property taxes section.
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjHouseholdRentOwnController), # Line 40b
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjHomeownerEligibilityController), # Line 40a
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjTenantEligibilityController), # Line 40a
```

Comment:
I appreciate the notes matching controllers to line numbers!

Reply:
I agree- can't take credit for it as I just copied from the ticket